src/main/scala/org/phenoscape/kb/Phenotype.scala

```diff
@@ -41,7 +41,7 @@ object Phenotype {
       eqs <- eqsFuture
       states <- statesFuture
       labelOpt <- labelFuture
-    } yield Phenotype(phenotype, labelOpt.map(_.label).getOrElse(""), states, eqs)
+    } yield Phenotype(phenotype, labelOpt.flatMap(_.label).getOrElse(""), states, eqs)
```

Comment:
Will `Phenotype` also necessarily have a `label` ?

Reply:
Currently it is required to. I didn't want to cascade changes too far and change that. Depending on where that case class is used we may want to make it optional down the road. But I thought it best to address that in response to an actual need.